docs/core/tutorials/netcore-hosting.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Hosting .NET Core
+description: Hosting the .NET Core runtime from native code 
+keywords: .NET, .NET Core, Hosting
+author: mikerou
+---
+
+# Hosting .NET Core
+
+Like all managed code, .NET Core applications are executed by a host. The host is responsible for starting the runtime (including components like the JIT and garbage collector), creating AppDomains, and invoking managed entry points.
+
+In most cases, .NET Core developers don't need to worry about hosting because .NET Core build processes provide a default host to run .NET Core applications. In some specialized circumstances, though, it can be useful to explicitly host the .NET Core runtime, either as a means of invoking managed code in a native process or in order to gain more control over how the runtime, itself, works.
+
+This article gives an overview of the steps necessary to start the .NET Core runtime from native code, create an initial AppDomain, and execute managed code in it.
+
+## Prerequisites
+
+Because hosts are, themselves, native applications, this tutorial will cover constructing a C++ application to host .NET Core. You will need a C++ development environment (such as that provided by [Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48146)).
+
+You will also want a simple .NET Core application to test the host with, so you should install the [.NET Core SDK and CLI](https://www.microsoft.com/net/core) and build a small .NET Core test app (such as a 'Hello World' app).
```

Comment:
SDK and CLI -> SDK 
(since the CLI comes with the SDK)

we also have topics that show you how to create that app in VS if you want to link to them. This is the VS 2015 version: https://github.com/dotnet/docs/blob/master/docs/csharp/getting-started/with-visual-studio.md